### ISSUE TITLE
active mode on Nav items in horizontal layout not working

### ISSUE THREAD

**Issue by rizkytegar** [zuramai/mazer#165]
is there any other idea, to add different color in active menu on horizontal Layout ?

![Layout Horizontal - Mazer Admin Dashboard](https://user-images.githubusercontent.com/55475891/195965816-f58cb86c-f145-4525-9a2c-e7210efde680.png)

it's not working

```
<li class="menu-item active">
  <a href="index.html" class="menu-link">
    <span><i class="bi bi-grid-fill"></i> Dashboard</span>
  </a>
</li>
```

**Comment by shreyamaurya029** [comment]
hey, i am interested in working on this issue. 
can u assign me this issue

**Comment by bagussatoto** [comment]
Hello @shreyamaurya029  thanks for visiting this project.

Please work on the development problem

**Comment by shreyamaurya029** [comment]
can u explain me more about the issue properly.

**Comment by gvnnunes** [comment]
i think he wants some highlighting on active menu-item, currently when adding .active class in horizontal menu doesn't highlight anything

**Comment by rizkytegar** [comment]
> i think he wants some highlighting on active menu-item, currently when adding .active class in horizontal menu doesn't highlight anything

yes bro.

**Comment by shreyamaurya029** [comment]
okay, thank you

**Comment by zuramai** [comment]
Fixed 7bafdb6

**Comment by rizkytegar** [comment]
very interesting, I will check it out
